packages/@glimmer/debug/lib/opcode-metadata.ts

```diff
@@ -1,1342 +1,1408 @@
 /* This file is generated by build/debug.js */
```

Comment:
I was just about to ask if this file was generated. Thanks, helpful comment :+1:

Reply:
It used to be generated but I need to update the generator 😀